CONTRIBUTING.md

```diff
@@ -167,16 +167,36 @@ squashed into multiple distinct change sets.
 
 ## Testing
 
-0. (temporary) get the latests integration tests
-   ```
-   flox develop .\#rust-env \
-     --override-input flox-floxpkgs-internal/flox github:flox/flox-bash-private
-   ```
-1. run cargo tests
-   - `cargo test` will run all pure rust tests tests
-   - `cargo test -F "extra-tests"` enables
-     unit tests that can't run in the nix sandbox (`-F impure-unit-tests`)
-     and integration tests that are shared with the bash implementation
-     of flox and ensure compatibility.
-   - `hivemind` watches the rust source code and continuously runs `cargo test -F "extra-tests"` on changes
-     can be configured through [./Procfile]
```

Comment:
We should still leave the instructions for those impure unit tests (and probably run those in CI as well).
Let's move that to a separate PR though, and get this one in first